Tests/SentryTests/SentryClientTests.swift

```diff
@@ -162,6 +162,30 @@ class SentryClientTest: XCTestCase {
             }
         }
     }
+    
+    func testCaptureEventTypeTransaction() {
```

Comment:
What about name it after what it does:

```suggestion
    func testCaptureEventTypeTransactionDoesNotIncludeThreadAndDebugMeta() {
```